<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardLettersDefPage</class>
 <widget class="QWizardPage" name="WizardLettersDefPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>497</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelDesc">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonLoadLetters">
       <property name="text">
        <string>_(&quot;Load letters from word list&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLoadFromDic">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>_(&quot;Use the values for the current dictionary for all the letters below (unknown letters will be ignored)&quot;)</string>
       </property>
       <property name="text">
        <string>_(&quot;Use values from current dictionary&quot;)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeLetters">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
